### ISSUE TITLE
2.0 ready

### ISSUE THREAD

**Issue by PrimozRome** [vuejs/awesome-vue#387]
Could we somehow mark all libraries, extensions and components which are 2.0 ready?

**Comment by posva** [comment]
This is a nice idea. However, we must make sure not to declare other libs as 2 incompatible (unless maintainers do not plan on upgrading) and state clearly the role of a 2.0 ready label. Maintainers of libs may easily forget about submitting a PR here.

**Comment by orenmizr** [comment]
also, some of the **tutorials** are 1.0 and there is a lot of interest regarding the 2.0. marking those as well in their own section would be helpful.

**Comment by pedro-mass** [comment]
Are all the tutorials listed 1.0? I tried looking for 2.0 tutorials, but have come up short.

**Comment by FranzSkuffka** [comment]
Bump.

So what are our options?
1. Explicit 2.0 compatible section / branch
2. Marking 2.0 compatibility / incompatibility if deprecated.

**Comment by rafaelspecta** [comment]
Any news on this? The list is updating very fast and it is getting confusing to find what is compatible with 2.0

**Comment by LinusBorg** [comment]
We are working on something else, a curated list with a dedicated website where we provide a curated selection of components from the community that are high-quality, actively maintained (or stable), with compatibility information etc.

Give a a couple of weeks to get this going.

**Comment by FranzSkuffka** [comment]
@LinusBorg don't you think that is overkill? I don't like the idea of 'elite' plugins. This place is super cool to just roam around and find something new. A website would destroy this. And introduce lots of overhead just for some prettyfication.

**Comment by LinusBorg** [comment]
@FranzSkuffka The new stuff won't replace awesome-vue. They fill different needs/purposes.

awesome-vue is a collection about everythign vue-related. IT's godd fore that pupose, but hard to maintain any kind of standard (compatibility, uptodate stars-ratings, does it have tests, is is actively developed?

Many people have proclaimed their need to find useful components/plugins quickly, and providing such an experience requires some level of curatition - and this is not possible with this simple awesome-vue markdown file.

**Comment by FranzSkuffka** [comment]
@LinusBorg I'm glad to hear this. To me it sounded like 'ooookey guys, awesome-vue is deprecated'.

Yet still, it appears that awesome-vue could use some structuring regarding the formatting of the entries, possibly even in a format that can be used in the project you brought up.

What do you think about using awesome-vue as a DB by extracting the entries from the markdown file?

**Comment by LinusBorg** [comment]
We will probably be using github as a DB for the curated list. We probably won't do that for awesome-vue because maintaining such info will probably be too much work. The Vue ecosystem is only beginning to grow and the list is already quite long.

But nonetheless, awesome-vue could use some love as well - but resources are scarce these days. We would be open to contributions regarding a cleanup / better structure :)

**Comment by ryandeussing** [comment]
There's a lot to be learned from the way the Ember community handles the issues of addon/plugin compatibility and discoverability. (I'm pretty sure the state of Vue 2.0 compatibility would be much higher if a similar resource existsed for Vue.)

1. [EmberObserver](https://emberobserver.com) is a canonical, complete list of libraries. It's where you look first.
2. Each library is assigned a [score](https://emberobserver.com/about)
3. Framework version [compatibility](https://emberobserver.com/addons/ember-try) is clearly visible

All of this is helpful, but the secret-sauce is the score, because it gives package authors just enough positive peer-pressure to take steps necessary to keep the score as high as possible:

```
- Is the source accessible?
- Is it more than an empty addon?
- Are there meaningful tests?
- Is the README filled out?
- Does the addon have a build?

SCORE

Every addon has a score, based on the following conditions:

1 point is given for every 'Yes' answer in the review.
1 point is given for a recent release (within last 3 months).
1 point is given for more than 1 commit within the last 3 months.
1 point is given for having more than 1 contributor.
1 point is given for having a download count in the last 30 days that is in the top 10% for all addons.
1 point is given for having a Github star count in the top 10% of all addons.
The total score is out of 10.
```
 
I'm optimistic based on the "Better Discoverability for the Ecosystem" section in @yyx990803's year-in-review post on Medium.

**Comment by FranzSkuffka** [comment]
@LinusBorg is it possible to contribute?

**Comment by LinusBorg** [comment]
Currently not, because we are still playing around with a few options to decide how to set this up from a technical pov. Once we have decided how to set everything up, the whole thing will be a public repo of course, and people are welcome to contribute. @Akryum do you agree?

**Comment by Akryum** [comment]
@LinusBorg Absolutely! When the thing will be more fleshed out, we will happily accept contributions.
